README.md

```diff
@@ -0,0 +1,83 @@
+# Oppiabot
+
+<p align="center">
+    <img src="images/oppiabot-display-image.png">
+</p>
+
+Oppiabot is a GitHub app built with [probot](https://github.com/probot/probot). It acts as a helper for the Oppia code respository to maintain the development workflow. It is hosted on [Heroku](https://www.heroku.com/).
+
+The bot currently performs the following functions:
+
+1. Checks whether a contributor creating a pull request has signed the CLA. It adds a label `Needs CLA` accordingly.
+
+2. Checks merge conflict and build status of a pull request and notifies the author as well as the maintainer-on-duty regarding the same.
+
+3. Closes stale pull requests.
+
+
+## Getting started
+
+Please refer to the following instructions to setup Oppiabot for the first time on your machine:
+
+1. Create a new, empty folder called `opensource/` within your home folder. Navigate to it (`cd opensource`), then [fork and clone](https://help.github.com/articles/fork-a-repo/) the Oppiabot repo. This will create a new folder named `opensource/oppiabot`. Navigate to `opensource/oppiabot/`.
+
+2. Install [Node.Js](https://nodejs.org/) and [NPM](https://www.npmjs.com/) on your machine using the following commands:
+
+  ```bash
+    sudo apt-get install nodejs
+    sudo apt-get install npm
+  ```
+
+3. Setup probot and other dependencies by running the following command:
+  ```bash
+    npm install
+  ```
+
+4. The Oppiabot uses environment variables. These are configured in the server settings. To deploy the bot locally, create a `.env` file and copy the contents of `.env.example` to it.
```

Comment:
Also mention that they will need to adjust the variables accordingly following the instructions in subsequent steps.

Reply:
Done, thanks!